Update user-agent-guidance.md for Windows 11 (#1474)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/user-agent-guidance.md

For more information, navigate to [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues].

> ### platformVersion Hint
> The Operating System version token in the `User-Agent` header has not been updated for Windows 11; it will still report `Windows NT 10.0`. 
> 
> To distinguish between Windows 10 and Windows 11, request the `platformVersion` client hint in Microsoft Edge version 95 or later. Values between and including `1.0.0` and `12.0.0` represent releases of Windows 10, while values of `14.0.0` and later represent releases of Windows 11.
> 

Reviewer: ```suggestion
### platformVersion Hint
The operating system version token in the `User-Agent` header has not been updated for Windows 11, and will still report `Windows NT 10.0`. 

To distinguish between Windows 10 and Windows 11, request the `platformVersion` client hint in Microsoft Edge version 95 or later. Values between and including `1.0.0` and `12.0.0` represent releases of Windows 10, while values of `14.0.0` or later represent releases of Windows 11.

```

For more information, navigate to [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues].

### platformVersion Hint

> The operating system version token in the `User-Agent` header has not been updated for Windows 11, and will still report `Windows NT 10.0`. 

Reviewer 2: ```suggestion
The operating system version token in the `User-Agent` header hasn't been updated for Windows 11, and still reports `Windows NT 10.0`. 

```
